processors/validator.py

```diff
@@ -6,13 +6,14 @@
 
 REQUIRED_FIELDS = [
     'award.awardingAgency.officeCode',
-    'award.actionDate'
+    'award.awardees.businessName',
+    'award.awardees.businessAddress',
     ]
 LENGTHS = [('award.awardingAgency.officeCode', 6)]
 NUMERIC_FIELDS = ['transaction.programActivity']
 POSSIBLE_VALUES = [
     ('transaction.objectClass', ['4110', '4101']),
```

Comment:
Is this list of possible object class codes just for the current pilot? There's a much longer list of possible object classes (which you link to in sba-business-rules.md). If this is just a subset for pilot purposes, I'd add a comment about that for the benefit of anyone looking at the source code.